packages/core/src/crypto/JwkTypes.ts

```diff
@@ -1,6 +1,157 @@
+import { KeyType } from './KeyType'
+
 export interface Jwk {
   kty: 'EC' | 'OKP'
-  crv: 'Ed25519' | 'X25519' | 'P-256' | 'P-384' | 'secp256k1'
+  crv: 'Ed25519' | 'X25519' | 'P-256' | 'P-384' | 'P-521' | 'Bls12381G1' | 'Bls12381G2'
```

Comment:
Personal preference: I'd instead define a separate union type with this amount of options. If possible, derived from the `KeyType` enum.

Reply:
I can extract into an union type. Can't derive it from KeyType, as the values are different. (everyone defines it a just a little bit dfiferent :) )

Reply:
If you mean that we're using different string values in AFJ, I'd say we need to make that consistent at some point. If you mean that other libs are using different values, I'd say we need to create transformer functions for those. Having multiple types that have the same semantic meaning, but are syntactically different just causes confusion, I think.

Reply:
Yes maybe. I wouldn't jump to use JWK as leading, but probably when we refactor the wallet interface and make it more generic, we can have better definitions of key types, algorithms etc..